### ISSUE TITLE
Include book name on header

### ISSUE THREAD

**Issue by abycurley** [WycliffeAssociates/DOC#117]

**Comment by linearcombination** [comment]
Headers are an automated feature that pick up HTML headline elements at a certain level, in this case the phrase 'Chapter X' where X is the current chapter on that particular page. What this means is that to include the book name with the chapter title we need to actually prepend the book name to the chapter title for the header to pick it up. We can do this if you like. The result would be chapter titles in the document itself that look like 'Mark Chapter 1' rather than 'Chapter 1' for instance. Please let me know if that is a change you would be happy with. We can do that if that is what you want.

**Comment by linearcombination** [comment]
@abycurley Did the comment above make sense? Given that comment do you want to leave it as currently is or have chapter titles in the document (and thus in the header) be of the form 'Mark Chapter 1' in order to achieve what you want? Could that work for you?

**Comment by abycurley** [comment]
Yes, but I have additional questions about naming each resource in the header. For instance, it would be nice if the header would indicate book, chapter, and resource type (e.g. Translation Notes, ULB). Does that throw a wrench in things?
